### DISCUSSION TITLE
[Help Wanted] How can I the default size of Codec in StreamingBody

### DISCUSSION THREAD

**Discussion by MMitsuha** [actix/actix-extras#491] [Q&A] [answered]
I have a very large packet (eg. 15 mb) to receive in websocket, But I noticed that the default size of each packet is 65536, which is defined in [this line](https://github.com/actix/actix-web/blob/8115c818c15792a6edab01f939f049b6c0e38792/actix-http/src/ws/codec.rs#L89), and there is no way I can modify it through [StreamingBody](https://github.com/actix/actix-extras/blob/18f94fa8b54453ac79cfe2ea04bcb8d7c10acb81/actix-ws/src/stream.rs#L39). How can I increase the size?

PS: I use this line to establish websocket connection:
```rust
let (response, mut session, mut stream) = actix_ws::handle(&req, body)?;
```

**Comment by robjtede** [comment, marked as answer]
Changing the max frame size is shown in this example: https://github.com/actix/actix-extras/blob/18f94fa8b54453ac79cfe2ea04bcb8d7c10acb81/actix-ws/examples/chat.rs#L71
